frontend_lib/src/component/OptionComponent/ArchiveDeleteContent.jsx

```diff
@@ -14,7 +14,7 @@ const ArchiveDeleteContent = props => {
 
   return (
     <div className='d-flex align-items-center'>
-      <button
+      {/* <button
```

Comment:
add the comment rules of algoo ("INFO - GB - data") and some explanation about why you are commenting this code with a reference to an issue or to something else